src/cmd/root.go

```diff
@@ -48,6 +49,10 @@ func Execute() {
 		doctor.DoctorCmd,
 	}
 
+	for _, command := range commands {
```

Comment:
Challenging my understanding here - please let me know where I could be off-base:

I believe this only needs to be set in the root command under `PersistentPreRun` such as:

```go
PersistentPreRun: func(cmd *cobra.Command, args []string) {
    log.SetOutput(os.Stderr)
    log.SetPrefix("go-oscal: ")
    if logFile != "" {
	file, err := logging.OpenLogFile(logFile)
	if err != nil {
	    log.Fatalf("failed to create log file: %s\n", err)
	}
	log.SetOutput(io.MultiWriter(os.Stderr, file))
    }
    cmd.SilenceUsage = true
},
```

Is there any benefit or draw-back to running it for each command individually?